### DISCUSSION TITLE
Quête d'aventurier

### DISCUSSION THREAD

**Discussion by Royal-91** [Crownicles/Crownicles#668] [Nouvelles idées]
Ca pourrait être sympa d'ajouter un système de quête.
Lorsque l'on arrive dans des grande ville comme **Claire de Ville** on reçoit un mini event qui nous propose une "quête". 
Cette quête nous demande d'aller dans un autre lieu de la map.
Lorsque l'on arrive à ce lieu, on tombe sur un mini event qui nous donne notre récompense. 

Ce système permettrait de renforcer le coté RP de nos personnages en tant qu'aventurier / chevalier.
Cela permettrait également de créer une source de revenue autre que de tout le temps volé les personnes. 
Ca permettrait également de donner un but défini aux joueurs qui le souhaite, ce qui les inciterai à jouer au jeu.

**Comment by niqore** [comment]
Ce sera fait dans la prochaine mise à jour (voir #153 )

**Reply by Royal-91** [reply]
Ah oui effectivement ça va être ajouté avec le système de Haut faits, je n'avais pas vu ^^'

**Comment by BastLast** [comment]
je me permet de lock la conversation du coup
